cli/popper/_version.py

```diff
@@ -0,0 +1 @@
+__popper_version__ = "2.5.1"
```

Comment:
this could be assigned to `0.0.0`, and then the tests we execute could check that the value is not `0.0.0` but instead is the value that `dunamai` returns.